terraform/domains/environment_domains/config/sandbox.tfvars.json

```diff
@@ -0,0 +1,20 @@
+{
+    "domains": ["apex"],
+    "cached_paths": ["/assets/*"],
+    "environment_short": "sb",
+    "environment_tag": "Sandbox",
+    "origin_hostname": "track-and-pay-sandbox.teacherservices.cloud",
+    "hosted_zone": {
+      "claim-funding-for-mentor-training.education.gov.uk": {
+        "resource_group_name": "s189p01-ittmsdomains-rg",
+        // value to be inputted after CR ?
+        "cnames": {
```

Comment:
cnames shouldn't be required. That was required when using cloudfront, but fd will create the cert validation/auth record